WordPress/src/main/java/org/wordpress/android/ui/jetpack/BackupAvailableItemsProvider.kt

```diff
@@ -0,0 +1,61 @@
+package org.wordpress.android.ui.jetpack
+
+import androidx.annotation.StringRes
+import org.wordpress.android.R.string
+import org.wordpress.android.ui.jetpack.BackupAvailableItemsProvider.BackupAvailableItemType.CONTENTS
+import org.wordpress.android.ui.jetpack.BackupAvailableItemsProvider.BackupAvailableItemType.MEDIA_UPLOADS
+import org.wordpress.android.ui.jetpack.BackupAvailableItemsProvider.BackupAvailableItemType.ROOTS
+import org.wordpress.android.ui.jetpack.BackupAvailableItemsProvider.BackupAvailableItemType.SQLS
+import org.wordpress.android.ui.jetpack.BackupAvailableItemsProvider.BackupAvailableItemType.THEMES
+import org.wordpress.android.ui.jetpack.BackupAvailableItemsProvider.BackupAvailableItemType.PLUGINS
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**
+ * This class provides the available item choices for Jetpack Backup Download
+ */
+@Singleton
+class BackupAvailableItemsProvider @Inject constructor() {
+    fun getAvailableItems(): List<BackupAvailableItem> {
+        return listOf(
+                BackupAvailableItem(
+                        THEMES,
+                        string.backup_item_themes
```

Comment:
Warning (⚠️ ): This is probably a wrong import. Consider using `R.string.xyz` instead.

PS: This applies to all the string resources in this provider.